builds/service-onboarding-build-pack/Jenkinsfile

```diff
@@ -167,30 +165,14 @@ node  {
 				checkout([$class: 'GitSCM', branches: [[name: '*/master']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[credentialsId: repo_credential_id, url: scmModule.getTemplateUrl(service_template)]]])
 				def config = LoadConfiguration()
 				def roleARN
-				def securityGroup
```

Comment:
are these not needed any more?

Reply:
Not during on-boarding user services. VPC support isn't available to the user services yet.